cpp/arcticdb/version/local_versioned_engine.cpp

```diff
@@ -1233,7 +1252,7 @@ std::pair<std::optional<VariantKey>, std::optional<google::protobuf::Any>> Local
     ) {
     auto version = get_version_to_read(stream_id, version_query);
     std::optional<AtomKey> key = version.has_value() ? std::make_optional<AtomKey>(version->key_) : std::nullopt;
-    return get_metadata(key).get();
+    return get_metadata(std::move(key)).get();
 }
 
 
```

Comment:
Since the compiler warning on line 1332 below was from one of your changes, do you mind fixing it?

Reply:
I didn't add this warning, but happy to fix it